File: coding-practice/50 Coding Challenges part 1/C#/problem8.cs
// coding challenge 8 : create a function that converts from Celsius to Fahrenheit
using System;
class problem8
{
    static float celsius_to_fahrenheit(float celsius){
        float fahrenheit = ((celsius*1.8f) + 32f);
        return fahrenheit;
    }
    static void Main(string[] args){
        float celsius = 37.5f;
        Console.Write(celsius + " Celsius is equal to " + celsius_to_fahrenheit(celsius) + " Fahrenheit");
    }
}

File: coding-practice/50 Coding Challenges part 1/JavaScript/scripts/problem8.js
// Coding challenge #7: Create a function that will convert from Celsius to Fahrenheit
function celsius_to_fahrenheit(celsius){
    fahrenheit = (celsius*(9/5)) + 32;
    return fahrenheit;
}

celsius = 33;
fahrenheit = celsius_to_fahrenheit(celsius);
document.write("Fahrenheit conversion of "+celsius+" Celsius is "+fahrenheit+" Fahrenheit");